scripts/aleth_with_log.sh

```diff
@@ -6,7 +6,7 @@ shift
 shift
 
 
-$ALETH_PATH $@ &> >(tail -n 10000 &> "$ALETH_TMP_OUT") &
+$ALETH_PATH $@ &> >(tail -n 100000 &> "$ALETH_TMP_OUT") &
```

Comment:
This does not have to be in here and actually the whole file could be removed now I guess...

Reply:
I'd keep it around in case we need the log again at some point